### PR TITLE
Calculator project

### PR DESCRIPTION
**Pull request by chuck032**
Completed basic/advanced calculator, the BMI calculator and I have attempted the travel calculator.

### CODE REVIEW

**Review by JSChilds — CHANGES_REQUESTED** [review]
The functionality works, but the actual user interaction is confusing. Firstly, your results are outputted to the console, which the user can't see - that's only for developers. You should use an alert or something similar to display the result of the calculation to the user. Also, because you've not put the bonus options in their own conditional statement, those prompts will always run regardless of what options the user picks, which is confusing.

**Comment by chuck032** [comment]
I've improved the user interaction with the calculator as well as cleaned up the DRY code. Hopefully I was able complete what you asked for.

File: calculator.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    console.log(value2);
+
+      //basic calculator
+      switch (choice) {
+
+        //addition
+        case "a":
+        case "add" :
+          var value3 = value1 + value2;
+          alert(value3);
```
Comment (JSChilds) [inline calculator.js]: This is still a little abrupt, I would at least include a message such as "The result is: " to make it a little more user friendly.

**Comment by chuck032** [comment]
I have made the calculator more user friendly now and also updated the readme file.

**Pull request merged**